internal/apiserver/ffi2swagger.go

```diff
@@ -193,6 +192,40 @@ func contractJSONSchema(ctx context.Context, params *fftypes.FFIParams, hasLocat
 	return openapi3.NewSchemaRef("", s), nil
 }
 
+/**
+ * Parse the FFI and build a corresponding JSON Schema to describe the response body for "invoke" or "query".
```

Comment:
I think this response body is only for query outputs

Reply:
Yep. Good catch. Misleading comments are the worst kind of comments.